pkg/services/accesscontrol/resourcepermissions/service.go

```diff
@@ -77,9 +79,36 @@ func New(cfg *setting.Cfg,
 		actions = append(actions, action)
 	}
 
+	// create the store for actionsets
+	log := log.New("accesscontrol.resourcepermissions")
+	actionSetsStore := NewInMemoryActionSets(log)
+
+	for permission, actions := range options.PermissionsToActions {
+		// for actionsets we need to know the resource
+		resource := ""
+		for _, a := range actions {
+
+			// assuming we get the ordering of this right,
+			// such as "dashboards", "folders", "teams"
+			if strings.Contains(a, ":") {
+				resource = strings.Split(a, ":")[0]
+				if resource != "folders" {
+					// currently only whitelisting folders
+					continue
```

Comment:
Folder permissions also grant access to resources within the folders, like dashboards, alerts etc, so we shouldn't filter based on the resource specified in the action.

In general, I feel like there's no harm in storing action sets for all resources. But if we really want to start with just folders, the check that you want is `options.Resource == "folder"`.

Reply:
@IevaVasiljeva should we store all action sets for all resources, but then when we start reading. We only do that for "dashboards" or a certain resource. We mentioned briefly that we should only do one resource for now.